lib/simplifyExpression/fractionsSearch/cancelLikeTerms.js

```diff
@@ -371,4 +377,41 @@ function isMultiplicationOfTerms(node) {
           !Node.PolynomialTerm.isPolynomialTerm(node));
 }
 
+function cancelCoeffs(numerator, denominator){
+  const denominatorTerm = new Node.PolynomialTerm(denominator);
+  const numeratorTerm = new Node.PolynomialTerm(numerator);
+
+  const denominatorCoeff = denominatorTerm.getCoeffNode();
+  const denomintorVariable = denominatorTerm.getSymbolNode();
+  const denominatorExponent = denominatorTerm.getExponentNode();
+
+  const numeratorCoeff = numeratorTerm.getCoeffNode();
+  const numeratorVariable = numeratorTerm.getSymbolNode();
+  const numeratorExponent = numeratorTerm.getExponentNode();
+
+  // simplify a constant fraction (e.g 2 / 4)
+  const frac = Node.Creator.operator('/', [numeratorCoeff, denominatorCoeff]);
+
+  let newCoeff = clone(denominatorCoeff);
```

Comment:
can you define this in the line before you use it? when I read it at first I was like wait what is this for

and if you return in like 399 you don't even need it

also can you name it `newDenominatorCoeff` to be more clear? then you don't have to do e.g. ` (the denominator coefficient)` later on in the comment :)